inspirehep/utils/other_conferences.py

```diff
@@ -0,0 +1,76 @@
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2015, 2016 CERN.
+#
+# INSPIRE is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#
+
+from invenio_search import current_search_client as es
+from jinja2 import render_template_to_string
+
+"""Conference export for single record in datatables format.
+:returns: list
+    List of lists where every item represents a datatables row.
+    A row consists of
+    [conference_name, conference_location, contributions, date]
+"""
+
+
+def conferences_in_the_same_series(recid, seriesname):
+
+        out = []
+        row = []
+
+        es_query = {
+            "filter": {
+                "bool": {
+                    "must": [
+                        {"match": {"series": seriesname}}
+                    ]
+                }
+            }
+        }
+
+        conferences_in_series = es.search(
+            index='records-conferences',
+            doc_type='conferences',
+            body=es_query,
+            _source=[
+                'control_number',
+                'title',
+                'place',
+                'opening_date',
+                'closing_date'
+            ]
+        )['hits']['hits']
```

Comment:
Maybe this function should be split in two, one doing the ES querying, one doing the rendering.